dev/reference/sql/statements/split-region.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Split Region
+summary: An overview of the usage of Split Region for the TiDB database.
+category: reference
+---
+
+# Split Region
+
+For each new table created in TiDB, one Region is segmented by default to store the data of this table. This default behavior is controlled by `split-table` in the configuration file. When the data in this Region exceeds the default Region size limit, the Region starts to split into two.
+
+In the above case, because there is only one Region at the beginning,  all write requests occur on the TiKV where the Region is located. If there are a large number of writes for the newly created table, hotspots are caused. 
```

Comment:
```suggestion
In the above case, because there is only one Region at the beginning, all write requests occur on the TiKV where the Region is located. If there are a large number of writes for the newly created table, hotspots are caused. 
```